nettacker/core/arg_parser.py

```diff
@@ -407,6 +407,14 @@ def add_arguments(self):
             default=Config.settings.compare_report_path_filename,
             help=_("compare_report_path_filename"),
         )
+        method_options.add_argument(
+            "-W",
+            "--wordlist",
+            action="store",
+            dest="user_wordlist",
+            default=Config.settings.user_wordlist,
```

Comment:
should be 'read_from_file' nor user_wordlist - there is no value in creating unnecessary setting/variable